packages/amplify-category-auth/provider-utils/awscloudformation/triggers/CreateAuthChallenge/boilerplate-create-challenge.js

```diff
@@ -1,6 +1,7 @@
 exports.handler = (event, context) => {
   if (event.request.session.length === 2 && event.request.challengeName === 'CUSTOM_CHALLENGE') {
     event.response.publicChallengeParameters = {};
+    event.response.publicChallengeParameters.trigger = true;
```

Comment:
In the ```captcha-create-challange.js``` file it is different:

```
event.response.publicChallengeParameters = { 'trigger' : 'true' };
```

It would be nice to have them the same way, whichever you pick, but I prefer this one in this PR.